runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -293,6 +363,12 @@ const_mcc_callPointPatching_unwrapper:
 	.align	3
 	.dword	mcc_callPointPatching_unwrapper
 
+L_commonLookupException:
```

Comment:
This needs to be reconciled with the version in PR #5432.

Reply:
Removed duplicated `L_commonLookupException`, and resolved merge conflicts.